redbot/cogs/mod/kickban.py

```diff
@@ -342,11 +349,18 @@ async def ban(
     ):
         """Ban a user from this server and optionally delete days of messages.
 
-        A user ID should be provided if the user is not a member of this server.
+        `days` is the amount of days of messages to cleanup on ban.
 
-        If days is not a number, it's treated as the first word of the reason.
+        Examples:
+           - `[p]ban 428675506947227648 7 Continue spam after told to stop.`
+            This ban Twentysix and it will delete 7 days worth messages.
+           - `[p]ban @Twentysix 7 Continue spam after told to stop.`
+            This ban Twentysix and it will delete 7 days worth messages.
```

Comment:
```suggestion
           - `[p]ban 428675506947227648 7 Continued to spam after told to stop.`
            This will ban Twentysix and it will delete 7 days worth of messages.
           - `[p]ban @Twentysix 7 Continued to spam after told to stop.`
            This will ban Twentysix and it will delete 7 days worth of messages.
```